lib/ansible/galaxy/collection/gpg.py

```diff
@@ -19,11 +19,12 @@
 
 if t.TYPE_CHECKING:
     from ansible.utils.display import Display
+    from ._types import DisplayQueueType, DisplayThreadProto
```

Comment:
*(no idea why I have `DisplayQueueType` here still; will probably have to drop it before merge)*
```suggestion
    from ._types import DisplayThreadProto
```